### PR TITLE
Add a blog post about GeoCAT tutorials

### PR DESCRIPTION
**Pull request by erogluorhan**
Adds the blog post along with its feature and content images.

### CODE REVIEW

**Comment by erogluorhan** [comment]
The feature image was created through my naive effort, so please feel free to suggest removal of replacement of that.

File: content/news/20221129-geocat_tutorials.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: "GeoCAT tutorials"
+author: "Orhan Eroglu"
+date: 2022-11-29T09:03:23-06:00
+type: news
+image: images/blog/20221129-geocat_tutorials.png
+feature_image: images/blog/20221129-geocat_tutorials-feature.png
+layout: staticpage
+---
+
+The [GeoCAT](https://geocat.ucar.edu/) team has launched the GeoCAT
```
Comment (jukent) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
The [GeoCAT](https://geocat.ucar.edu/) team has launched "GeoCAT 
```

File: content/news/20221129-geocat_tutorials.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+author: "Orhan Eroglu"
+date: 2022-11-29T09:03:23-06:00
+type: news
+image: images/blog/20221129-geocat_tutorials.png
+feature_image: images/blog/20221129-geocat_tutorials-feature.png
+layout: staticpage
+---
+
+The [GeoCAT](https://geocat.ucar.edu/) team has launched the GeoCAT 
+tutorials to provide its community with guided teaching as well as
```
Comment (jukent) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
Tutorials" to provide its community with guided teaching as well as 
```

File: content/news/20221129-geocat_tutorials.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+feature_image: images/blog/20221129-geocat_tutorials-feature.png
+layout: staticpage
+---
+
+The [GeoCAT](https://geocat.ucar.edu/) team has launched the GeoCAT 
+tutorials to provide its community with guided teaching as well as 
+self-taught content.
+
+The tutorial materials can be accessed via its [GitHub 
+repository](https://github.com/NCAR/geocat-tutorials); cloning and
```
Comment (jukent) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
repository](https://github.com/NCAR/geocat-tutorials); by cloning and 
```

File: content/news/20221129-geocat_tutorials.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+The tutorial materials can be accessed via its [GitHub 
+repository](https://github.com/NCAR/geocat-tutorials); cloning and 
+setting up a local configuration as described in the repository home, 
+or through its [Binder 
+environment](https://mybinder.org/v2/gh/NCAR/geocat-tutorials/main) 
+right away online! 
+
+The content can be viewed at its statically generated [documentation 
+page](https://geocat-tutorials.readthedocs.io/en/latest/) as well. The
```
Comment (jukent) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
page](https://geocat-tutorials.readthedocs.io/en/latest/) as well.
```

File: content/news/20221129-geocat_tutorials.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+The tutorial materials can be accessed via its [GitHub 
+repository](https://github.com/NCAR/geocat-tutorials); cloning and 
+setting up a local configuration as described in the repository home, 
+or through its [Binder 
+environment](https://mybinder.org/v2/gh/NCAR/geocat-tutorials/main) 
+right away online! 
+
+The content can be viewed at its statically generated [documentation 
+page](https://geocat-tutorials.readthedocs.io/en/latest/) as well. The 
+readers of this documentation page can also access to the past tutorial
```
Comment (jukent) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
Readers of this documentation page can also access past tutorial 
```

**Review by jukent — COMMENTED** [review]
Looks great! Thanks for putting this together. I made a few minor suggestions for grammar.

File: content/news/20221129-geocat_tutorials.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+image: images/blog/20221129-geocat_tutorials.png
+feature_image: images/blog/20221129-geocat_tutorials-feature.png
+layout: staticpage
+---
+
+The [GeoCAT](https://geocat.ucar.edu/) team has launched the GeoCAT 
+tutorials to provide its community with guided teaching as well as 
+self-taught content.
+
+The tutorial materials can be accessed via its [GitHub
```
Comment (anissa111) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
The tutorial materials can be accessed via [GitHub 
```

File: content/news/20221129-geocat_tutorials.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+The [GeoCAT](https://geocat.ucar.edu/) team has launched the GeoCAT 
+tutorials to provide its community with guided teaching as well as 
+self-taught content.
+
+The tutorial materials can be accessed via its [GitHub 
+repository](https://github.com/NCAR/geocat-tutorials); cloning and 
+setting up a local configuration as described in the repository home, 
+or through its [Binder
```
Comment (anissa111) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
or through [Binder 
```

File: content/news/20221129-geocat_tutorials.md (hunk 4 above)
Comment (anissa111) [inline content/news/20221129-geocat_tutorials.md]:
```suggestion
The content can be viewed as a statically generated [documentation 
page](https://geocat-tutorials.readthedocs.io/en/latest/) as well. The 
```

**Review by anissa111 — COMMENTED** [review]
Definitely could be a me issue, but my local build isn't showing the feature image: 
image

Other than that, I just left a couple tiny suggestions!

File: content/news/20221129-geocat_tutorials.md (hunk 1 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

File: content/news/20221129-geocat_tutorials.md (hunk 2 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

File: content/news/20221129-geocat_tutorials.md (hunk 3 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

File: content/news/20221129-geocat_tutorials.md (hunk 4 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

File: content/news/20221129-geocat_tutorials.md (hunk 5 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

File: content/news/20221129-geocat_tutorials.md (hunk 4 above)
Comment (erogluorhan) [reply content/news/20221129-geocat_tutorials.md]: Done

**Comment by erogluorhan** [comment]
> Definitely could be a me issue, but my local build isn't showing the feature image
> 

No, it was due to an error with the mis-location of the feature image. Fixed it now. Thanks!

**Review by anissa111 — APPROVED** [review]

**Pull request merged**